### PR TITLE
Feat/submodules

### PR DESCRIPTION
**Pull request by adatzer**
A preview of the final outcome after merging and running the `submodules_update` github action can be seen in this temporary fork [here](https://github.com/adatzer/iglu).

### CODE REVIEW

**Comment by snowplowcla** [comment]
Thanks for your pull request. Is this your first contribution to a Snowplow open source project? Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

:memo: Please visit https://docs.snowplowanalytics.com/docs/contributing/contributor-license-agreement/ to learn more and sign.

Once you've signed, please reply here (e.g. I signed it!) and we'll verify. Thanks.

File: README.md [hunk 1]
```diff
@@ -1,36 +1,79 @@
...
+[![License][license-image]][license]
+[![Discourse posts][discourse-image]][discourse]
 
-Iglu is a machine-readable, open-source schema repository for **[JSON Schema][json-schema]** from the team at **[Snowplow Analytics][snowplow-website]**. A schema repository (sometimes called a registry) is like npm or Maven or git, but holds data schemas instead of software or code.
+[![Snowplow logo][logo-image]][website]
 
-Iglu is used extensively in **[Snowplow][snowplow-repo]**. For a presentation on how we came to build Iglu, see **[this blog post][snowplow-schema-post]**.
+## Overview
+
+Iglu is a machine-readable, open-source schema repository for **[JSON Schema][json-schema]** from the team at **[Snowplow Analytics][website]**. A schema repository (also called a registry) is like npm or Maven or git, but holds data schemas instead of software or code.
```
Comment (paulboocock) [inline README.md:11]: I'd suggest dropping "Analytics" from "Snowplow Analytics" since we're heading that way with everything else now.

**Review by paulboocock — APPROVED** [review]
There's also now a helm chart for iglu servers we could include in the infra folder: https://github.com/snowplow-devops/helm-charts/tree/main/charts/snowplow-iglu-server

However, its a sub folder rather than a repository on its own so not sure how well it'll integrate here...

Other than that, this looks good. I imagine @stanch might want some more Emojis on the root README though, but I'll leave that up to him ;-)

**Comment by adatzer** [comment]
Thank you for reviewing @paulboocock !
Concerning the snowplow-iglu server helm chart, it would be great to reference as well. Indeed, though, submodules need to be git repositories, so we cannot add only the relevant subdirectory as a submodule. The alternatives i have considered so far are:

- Add the whole helm-charts repository as submodule under `3-infrastructure` path, and make it update only on an `snowplow-iglu-server/x.x.x` release tag pattern. And in the [readme table](https://github.com/snowplow/iglu/blob/feat/submodules/3-infrastructure/README.md) link to the relevant subdirectory.
- Add the relevant [subdirectory](https://github.com/snowplow-devops/helm-charts/tree/main/charts/snowplow-iglu-server) as a simple directory under `3-infrastructure` (not as a submodule). Then extend the `submodules_update` github actions workflow to make a `diff` and apply it as a `patch`, again on relevant release tag discovery.
- Only add another entry to the `3-infrastructure`'s Readme table.

Each of the above has its pros and cons and i am not sure which makes more sense. What do you think?

**Comment by adatzer** [comment]
The rebase force-pushed includes:

1. Reorder of commits to allow setting the new url for `igluctl` and `iglu-server`
2. Switch to snowplow.io domain (included in ["Update Readme files" commit](https://github.com/snowplow/iglu/pull/528/commits/4306e750e1235289f5c14b5f89e50cab8db7e8b0))
3. Cherry-pick of commit from https://github.com/snowplow/iglu/pull/483
4. Issue https://github.com/snowplow/iglu/issues/529

**Pull request merged**